linera-views/src/views/bucket_queue_view.rs

```diff
@@ -55,9 +55,9 @@ enum KeyTag {
 /// The `StoredIndices` contains the description of the stored buckets.
 #[derive(Clone, Debug, Default, Serialize, Deserialize)]
 struct StoredIndices {
-    /// The stored buckets with the first index being the size at most N and the
-    /// second one is the index. If the index is 0 then it correspond to the Front,
-    /// otherwise to the Index.
+    /// The stored buckets with the first index being the size (at most N) and the
```

Comment:
"with the first value"? (or component)
why the capital letters "Front" and "Index"?

Reply:
It corresponded to the items in the `KeyTag`. I have updated the documentation.